app/integrations/google_workspace/google_service.py

```diff
@@ -87,6 +87,7 @@ def wrapper(*args, **kwargs):
                     logging.warning(
                         f"Unsupported parameters in '{func.__name__}' were filtered out: {', '.join(unsupported_params)}"
                     )
+                return result
```

Comment:
I'll do some other tests but I recall getting inconsistent results earlier with my unit tests. Probably not needed because I initially didn't have it but I'll check it out when I get back!

Reply:
OK, I updated the function as it doesn't require the second return statement. I think it's a residual from prior to assessing if the returned results were a tuple. the tests all pass now!